### PR TITLE
testing/colormake: new aport

### PR DESCRIPTION
**Pull request by HRio**

### CODE REVIEW

File: testing/colormake/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# Maintainer: Henrik Riomar <henrik.riomar@gmail.com>
+pkgname=colormake
+pkgver=0.9.20140503
+pkgrel=0
+pkgdesc="A simple wrapper around make to colorize the output"
+url="http://bre.klaki.net/programs/colormake"
+arch="noarch"
+license="GPL2+"
+depends="perl"
+options="!check"
```
Comment (Shizmob) [inline testing/colormake/APKBUILD]: Is the upstream test suite broken or otherwise unsuitable? We'd like `check()` on as many package as possible. :)

Comment (HRio) [inline testing/colormake/APKBUILD]: does not contain any checks, will add "PATH=.:${PATH} ./colormake --version" as check(), is that OK?

Comment (Shizmob) [inline testing/colormake/APKBUILD]: I'm not sure if the $PATH part is needed, but yes, running the binary is an okay smoke test. :)

**Comment by HRio** [comment]
@Shizmob bash missing from depends and make from checkdepends, fixed in 63ae8ae

**Pull request closed without merging**